### PR TITLE
(fix) Talk Details screen

### PR DESCRIPTION
**Pull request by mazenchami**
# Description

[Trello Card](172-when-data-is-missing-add-coming-soon-dont-leave-empty) — #136 
[Trello Card](171-sneaky-data-issues-missing-social-links) — #137 

- making sure talk screens have a "coming soon" label if empty
- updating how we're displaying socials

# Graphics

| iOS            | Android            |
| -------------- | ------------------ |
|  |  |

# Checklist:

- [x] I have done a thorough self-review of my code
- [ ] I have tested with a screen reader and font-scaling turned on and added necessary accessibility features
- [x] I have run tests and linter

### CODE REVIEW

File: app/components/SocialButton.tsx [hunk 1]
```diff
@@ -40,6 +41,18 @@ export function SocialButton(props: SocialButtonProps) {
   )
 }
 
+interface SocialButtonsProps {
+  socialButtons: Array<SocialButtonProps>
+}
+
+export const SocialButtons = ({ socialButtons }: SocialButtonsProps) => (
+  <>
+    {socialButtons.map((socialButtonProps) => (
```
Comment (morganick) [inline app/components/SocialButton.tsx:50]: In the cases where we don't have a URL, should we filter those out?

Comment (mazenchami) [reply app/components/SocialButton.tsx:50]: that happens above on line 29. didn't want to replicate it

Comment (mazenchami) [reply app/components/SocialButton.tsx]: my bad, the `key` should be `${icon}-${index}`. updated here: https://github.com/infinitered/ChainReactApp2023/pull/151/commits/fbcadd2d14fd06e79d18448f6a00ac5a8b066acd

**Review by morganick — APPROVED** [review]
LGTM 👍

File: app/screens/TalkDetailsScreen/TalkDetailsScreen.tsx [hunk 2]
```diff
@@ -74,24 +79,31 @@ const SCREEN_WIDTH = Dimensions.get("screen").width
...
+    str && str.length > 1 ? str : translate("talkDetailsScreen.comingSoon")
+
   return {
     title: talk?.name,
     subtitle: `${formatDate(schedule["day-time"], "MMMM dd, h:mmaaa")} PT`,
     imageUrl: talk?.["speaker-s"][0]?.["speaker-photo"].url,
     fullName: talk?.["speaker-s"][0]?.name,
     company: talk?.["speaker-s"][0]?.company,
-    description: talk?.description,
+    description: stringOrPlaceholder(talk?.description),
```
[the remark this answers is not in the document]
Comment (mazenchami) [reply app/screens/TalkDetailsScreen/TalkDetailsScreen.tsx]: Not a bad idea. implemented here: https://github.com/infinitered/ChainReactApp2023/pull/151/commits/13a8b6ee3c2c92a2e6a84839ad726432e975a210

**Pull request merged**